libr/core/cconfig.c

```diff
@@ -3534,6 +3568,16 @@ R_API int r_core_config_init(RCore *core) {
 	SETBPREF ("esil.nonull", "false", "Prevent memory read, memory write at null pointer");
 	SETCB ("esil.mdev.range", "", &cb_mdevrange, "Specify a range of memory to be handled by cmd.esil.mdev");
 
+	/* json encodings */
+	n = NODECB ("json.encoding.strings", "none", &cb_jsonencoding);
```

Comment:
there's no root `json` config variable, so i would suggest using `cfg.json.str` and `cfg.json.num`. it's shorter and fits into `cfg`. no need to highlight the "encoding" thing imho